Network/Sendfile/Fallback.hs

```diff
@@ -26,6 +29,9 @@ sendfile sock path EntireFile hook =
 sendfile sock path (PartOfFile off len) hook =
     runResourceT $ EB.sourceFileRange path (Just off) (Just len) $$ sinkSocket sock hook
 
+sendfileFd :: Socket -> Fd -> FileRange -> IO () -> IO ()
+sendfileFd _sock _fd _range _hook = undefined
```

Comment:
I believe that should be possible. This is just a stand in. undefined has the nice property that it will blow up if called. I wonder if we could implement `sendfileFd` in a meaningful, yet broken form?

Reply:
I misunderstood and you are right about `undefined`. But is it possible to use `return ()` instead?